### DISCUSSION TITLE
Preset

### DISCUSSION THREAD

**Discussion by Martysanti** [publicsamples/Quetzalcoatl#21] [General]
Hi everybody, 

I see a video from a youtuber about this great plugin and he have a lot of preset very cool. And after install the plug, i don't have the same preset and the sample. So i think i don't have the good folder or anyelse ? What the solution ? I'm on Mac M2 

Thank you all

**Comment by RickTaylor79** [comment]
Hmm, the latest version (0.7.0) has more presets than any previous release, make sure you have this installed. Did you install the expansion packs? this will increase number of available sounds and presets..

**Reply by k-srvHs** [reply]
I also watched a video where there was a Roland Misc pack but I don't see it as an option on my expansion packs. I also wanted to purchase the expanded version but I only have $11 on my Cashapp which my friend from Texas sent me. I'm Kenyan. The 10.50 euros/pounds converts to $16.09...*tears*
<img width="931" alt="Screenshot 2024-08-12 at 19 28 24" src="https://github.com/user-attachments/assets/68ae8d03-9704-4ae2-8ae6-a623c35b296b">
<img width="847" alt="Screenshot 2024-08-12 at 19 29 30" src="https://github.com/user-attachments/assets/ecea17ab-cead-4e0b-8b49-65bd5769d317">

**Comment by RickTaylor79** [comment]
Yeah, thats part of the expanded version. Here's a code for 50% off:

https://modularsamples.gumroad.com/p/it-s-the-first-annual-modulasample-com-summer-sale-50-off-everything

Alternatively, if you'd be willing to make some presets for it I'll sort
you out a free copy + some bonus stuff. Email me for details.

**Reply by k-srvHs** [reply]
I'm a NOOB on this platform. Since you didn't include your email address I'm assuming I'm supposed to click somewhere to automatically see it? All I would need is to learn how to create the presets and I'll most definitely experiment. I truly appreciate this quick response.

**Reply by RickTaylor79** [reply]
You can contact me at modularsamples@gmail.com.

**Reply by k-srvHs** [reply]
Sent!
